modules/android/vsandroid_androidproj.lua

```diff
@@ -80,10 +80,18 @@
 	end
 
 
+	function android.androidAPILevel(cfg)
```

Comment:
I'm pretty sure you can just call the function from the other file without duplicating the code in this file. The `android` variable is the same in both files, so it should work without any other modifications.

Reply:
Sure, I'll fix this.